domain/lib/src/usecases/audio_recorder/audio_recorder_viewstate.dart

```diff
@@ -0,0 +1,40 @@
+
+import 'package:domain/domain.dart';
+
+class AudioRecorderSuccessViewState extends Success {
+  final List<FileInfo> file;
+  AudioRecorderSuccessViewState(this.file);
+
+  @override
+  List<Object> get props => [file];
+}
+
+class AudioRecorderStarted extends Success {
+  @override
+  List<Object> get props => [];
+}
```

Comment:
```suggestion
class AudioRecorderStarted extends Success {}
```

Reply:
Success is an abstract class that extends equatable so this override cant be removed